libs/coin-modules/coin-casper/src/api/index.ts

```diff
@@ -146,10 +152,23 @@ export const fetchNetworkStatus = async (): Promise<NNetworkStatusResponse> => {
 };
 
 export const fetchTxs = async (addr: string): Promise<ITxnHistoryData[]> => {
-  const response = await casperIndexerWrapper<ITxnHistoryData>(
-    `/accounts/${addr}/ledgerlive-deploys`,
+  let page = 1;
+  const res: ITxnHistoryData[] = [];
+  const limit = 100;
+
+  let response = await casperIndexerWrapper<ITxnHistoryData>(
+    `/accounts/${addr}/ledgerlive-deploys?limit=${limit}&page=${page}`,
   );
-  return response;
+  res.push(...response.data);
+
+  while (response.pageCount > page) {
+    page++;
+    response = await casperIndexerWrapper<ITxnHistoryData>(
+      `/accounts/${addr}/ledgerlive-deploys?limit=${limit}&page=${page}`,
+    );
+    res.push(...response.data);
```

Comment:
We could probably use concat here instead of spreading the array it's usually faster in JS AFAIK
https://www.measurethat.net/Benchmarks/ShowResult/590296

Reply:
Fixed! Please have another look 🙏🏻